robolectric-resources/src/main/java/org/robolectric/RuntimeEnvironment.java

```diff
@@ -28,7 +28,8 @@ public static RobolectricPackageManager getRobolectricPackageManager() {
     }
 
     public static void setRobolectricPackageManager(RobolectricPackageManager newPackageManager) {
-        packageManager = newPackageManager;
+      if (packageManager != null) packageManager.reset();
```

Comment:
Not sure what the official policy is, I'm still very much following Square style. @erd